### PR TITLE
Release version 3.4.3

### PR DESCRIPTION
**Pull request by imbyungjun**
- Change ngrinder version to 3.4.3
- This release fixes log rolling error and xss filter error

### CODE REVIEW

**Comment by donggyu04** [comment]
Can one of the admins verify this patch?

File: RELEASE-NOTE.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
+3.4.3 (2019.07.09)
+==================
+> nGrinder 3.4.3 
+ is the bug fix and minor enhancement release.
+
+- Changes
+  * Make agent worker log rolling work correctly.
```
Comment (junoyoon) [inline RELEASE-NOTE.md]: log-rotation

File: RELEASE-NOTE.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+    - Allow to delete log rolling backup file on secure mode
```
Comment (junoyoon) [inline RELEASE-NOTE.md]: delete above

File: RELEASE-NOTE.md [hunk 3]
```diff
@@ -1,3 +1,22 @@
...
+  * Make script with html escape chracter be saved correctly.
+    - Bump Lucy XSS filter version up to 2.0.1
+    - Show script contents without HTML escape characters
+ 
+- Bug Fix
+  * #363 ValidateScript error
+  * #385 HTML escape char cover error
+  * #387 about groove Script Error
+  * #389 `&quot;` error
+  * #433 Agent worker log isn't rolling on secure mode
```
Comment (junoyoon) [inline RELEASE-NOTE.md]: Add fix for each entries

**Pull request merged**